arch/s390/slide_hash_vx.c

```
@@ -0,0 +1,35 @@
+/* slide_hash_vx.c - VX version of slide_hash for S390 processors
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#ifdef S390_VX
+
+#include <vecintrin.h>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)